controller/search_blackbox_test.go

```diff
@@ -95,6 +95,7 @@ func (s *searchControllerTestSuite) TestSearchWorkItems() {
 	require.NotEmpty(s.T(), sr.Data)
 	r := sr.Data[0]
 	assert.Equal(s.T(), q, r.Attributes[workitem.SystemTitle])
+	assert.Equal(s.T(), fxt.WorkItems[0].Fields[workitem.SystemNumber], r.Attributes[workitem.SystemNumber])
```

Comment:
I really think this should say: 

```go
assert.Equal(s.T(), fxt.WorkItems[0].Number, r.Attributes[workitem.SystemNumber])
```

instead.

Reply:
Fixed d991bc823d20c8d41cef0b0479d858133300277a